test/powershell/Modules/Microsoft.PowerShell.Utility/New-Object.Tests.ps1

```diff
@@ -164,6 +262,38 @@ Describe "New-Object DRT basic functionality" -Tags "CI" {
 		$result = New-Object -TypeName PSObject -property @{foo=123}
 		$result.foo | Should Be 123
 	}
+
+    It "New-Object object cannot create COM object under constrained mode" -skip:(-not $IsWindows) {
```

Comment:
I think we should add these tests here: https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1#L226